public/app.js

```diff
@@ -0,0 +1,330 @@
+// AUTH CONTROLLER
+var authController = (function() {
+    return {
+        //Authentication state listener
+        initApp: function() {
+            firebase.auth().onAuthStateChanged(function (user) {
+                if (user) {
+                    console.log('Login Successful');
+                    console.log("Attempting to bind: " + user.email)
+                    console.log('Bind Successful');
```

Comment:
I'm a bit confused as to what's going on here. You say you're attempted to bind a user than report that bind as successful without actually doing anything

Reply:
resolved